torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -918,6 +918,17 @@ ExprHandle TensorExprKernel::getStrideArg(
   return it->second;
 }
 
+std::vector<torch::jit::StrideInput>& TensorExprKernel::
+    getSymbolicInputStrideDesc(const torch::jit::Value* value) {
+  for (size_t i : c10::irange(graph_->inputs().size())) {
+    if (value == graph_->inputs().at(i)) {
```

Comment:
Iterating over the list of inputs here everytime might affect perf. But I don't see a way around it, if we don't want to retain the input map.